draft-packages/modal/KaizenDraft/Modal/Primitives/ModalAccessibleLabel.scss

```diff
@@ -1,8 +1,36 @@
 @import "~@kaizen/design-tokens/sass/color";
 @import "~@kaizen/design-tokens/sass/border";
-.label {
+@import "~@kaizen/design-tokens/sass/layout";
+
+//TODO - Remove below styles for Heading once responsive styling has been added to the Heading Component
+$ca-breakpoint-small-mobile: 375px;
+
+@mixin ca-media-small-mobile {
+  @media (max-width: #{$ca-breakpoint-small-mobile}) {
+    @content;
+  }
+}
+
+.modalLabel {
   position: relative;
 
+  //TODO - Remove below styles for Heading once responsive styling has been added to the Heading Component
+  h2 {
+    @media (max-width: $layout-breakpoints-medium) {
+      font-size: 1.625rem;
```

Comment:
I am still trying to find out if our best practice should be to always use `rem` for A11y (allegedly it's better for scaling content consistently), but I definitely find that when using `rem` for `font-size` it should also be used for the accompanying `line-height`.